test/integration/sequelize/query.test.js

```diff
@@ -342,6 +342,20 @@ describe(Support.getTestDialectTeaser('Sequelize'), () => {
           await this.UserVisit.sync({ force: true });
         });
 
+        it('emits raw errors if requested', async function() {
+          const sql = 'SELECT 1 FROM NotFoundTable';
+          try {
+            await this.sequelize.query(sql, { rawErrors: false });
+          } catch (err) {
+            expect(err).to.be.instanceOf(DatabaseError);
+          }
+          try {
+            await this.sequelize.query(sql, { rawErrors: true });
+          } catch (err) {
+            expect(err).to.not.be.instanceOf(DatabaseError);
```

Comment:
Can you please assert against the actual error?

Reply:
The actual error is something thrown deep within the connection/database library, which is dialect specific I think.

Reply:
alright!